src/components/banner/banner.import.less

```diff
@@ -0,0 +1,74 @@
+.head-image {
```

Comment:
@urvi-pisal I realize these are likely copied from the original CSS, but you should nest them in typical LESS fashion - they will be more legible and maintainable. 

```
.head-image {
  ...
  .head-image-content {
     ...
  }
}
```

Also, to fully follow BEM naming convention, `.head-image-content` should be changed to `.head-image__content`